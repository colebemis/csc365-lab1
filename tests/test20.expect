Enter query: 4618928770913378229; 2.85; 1; FALKER, ADOLPH; 54
9066073379779809574; 3.14; 1; FALKER, ADOLPH; 0
Enter query: 