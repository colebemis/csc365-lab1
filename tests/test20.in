RAW: GRADE=1
Q
