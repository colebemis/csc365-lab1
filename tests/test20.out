Enter query: 5437756226515167994; 2.85; 1; FALKER, ADOLPH; 54
8685481672206056646; 3.14; 1; FALKER, ADOLPH; 0
Enter query: 